lib/iris/util.py

```diff
@@ -1211,6 +1211,43 @@ def add_coord(coord):
     return new_cube
 
 
+def cube_squeeze(cube):
+    """
+    Removes any dimension of length one. If it has an associated DimCoord or
+    AuxCoord, this becomes a scalar coord.
+
+    Args:
+
+    * src_cube (:class:`iris.cube.Cube`)
+        Source cube to remove length 1 dimension(s) from.
+
+    Returns:
+        A new :class:`iris.cube.Cube` instance without any dimensions of
+        length 1.
+
+    For example::
+
+        >>> cube.shape
+        (1, 360, 360)
+        >>> ncube = iris.util.cube_squeeze(cube)
+        >>> ncube.shape
+        (360, 360)
+
+    .. warning::
+
+        Calling this method will trigger any deferred loading, causing the
```

Comment:
Is that true? I thought cube indexing preserved the deferred state of the data within.